wrappers/services.go

```diff
@@ -377,7 +383,7 @@ Requires={{.MountUnit}}
 Wants={{.PrerequisiteTarget}}
 After={{.MountUnit}} {{.PrerequisiteTarget}}{{range .After}} {{.}}{{end}}
 {{- if .Before}}
-Before={{ range .Before -}}{{.}} {{- end}}
+Before={{range .Before}}{{.}} {{end}}
```

Comment:
What is the meaning of the dashes there?

Reply:
Go template notation, `-` trim all whitespace before/after the `{{tag}}`. Since newline counts as whitespace too, you can easily end up with something totally wrong :confused:. Personally, I find it super confusing.

Reply:
Yeah, this is a bugfix, maybe I should roll this into its own PR. In any case, its not clear to me how to solve this using the template language without extra whitespace, essenially I need strings.Join(before, " ") inside the template.

Reply:
https://play.golang.org/p/QERhAshfyMC best I could come up with